asus/zephyrus/ga403/default.nix

```diff
@@ -0,0 +1,93 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+let
+  inherit (lib)
+    mkDefault
+    mkForce
+    ;
+in
+{
+
+  imports = [
+    ../../../common/cpu/amd
+    ../../../common/cpu/amd/pstate.nix
+    ../../../common/gpu/amd
+    ../../../common/pc/laptop
+    ../../../common/pc/ssd
+    ../../../common/gpu/nvidia/prime.nix
+    ../../../common/gpu/nvidia/ada-lovelace
+  ];
+
+  config = {
+    # Configure basic system settings:
+    boot = {
+      # Note: it seems being on the latest kernel is important for CPPC to work, which is required for the amd_pstate
+      # https://bugzilla.kernel.org/show_bug.cgi?id=218686
+      kernelPackages = mkDefault pkgs.linuxPackages_latest;
```

Comment:
Please set a minimum kernel version instead:

```
kernelPackages = lib.mkIf (lib.versionOlder pkgs.linux.version "6.X") pkgs.linuxPackages_latest;
```

Replace `x` with the kernel version that works for you. In doubt the current latest kernel version if you don't want to test others.